src/components/auth/login/Login.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Login.scss';
+import '../../../assets/scss/Login.scss';
+import '../../../assets/scss/SocialLogin.scss';
+import Loading from '../../common/Loading';
+
+import FacebookOption from '../../social/facebook';
+import GoogleOption from '../../social/google';
+
+export default function Login(props) {
+  const { handleSubmit, onInputChange, isLoading } = props;
+  return (
+    <div className="login-clean">
+      <form onSubmit={handleSubmit}>
+        <h2 className="sr-only">Login Form</h2>
+        <div className="illustration">
+          <i className="icon ion-ios-navigate" />
+        </div>
+        <div className="social-apps" />
+        <div className="text-center social-btn">
+          <FacebookOption />
+          <br />
+          <GoogleOption />
+          <br />
+        </div>
+
+
```

Comment:
@Manorlds-Eaglespark can we have an `OR` separator here to give visual feedback that one has an option of either logging in with social media authentication or our own application authentication.

Reply:
Noted. Thanks.